2.1/2.1.1/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+<script>
+alert('Я JavaScript!')
```

Comment:
";" should be placed after each instruction.
Please read the following paragraph:
![image](https://github.com/tacya9/JS-1-2/assets/115877101/b194c7d7-ba09-430d-b825-396a14eae3a9)